backend/package.json

```diff
@@ -33,10 +33,12 @@
         "@fastify/cors": "8.2.1",
         "@fastify/swagger": "8.3.1",
         "@fastify/swagger-ui": "1.4.0",
+        "axios": "1.3.4",
```

Comment:
I think we can do it without axios

Reply:
Resolved in [2253ca5](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/314/commits/2253ca5bf25402433bc32c8786d726d6d4602dd3)